### PR TITLE
Add the release note for 1.20.2

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: site/src/pages/release-notes/1.20.2.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2022-11-07
+---
+
+## 🛠️ Bug fixes
+
+- You no longer see `StacklessClosedChannelException` when when an incoming HTTP/1.0 or HTTP/1.1 request has
+  a 'Connection: close` header. #4508
+- An <type://HttpRequest> is not failed with the <type://AbortedStreamException> anymore
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.20.2.mdx:9]:
```suggestion
- An <type://HttpRequest> is not failed with <type://AbortedStreamException> anymore
```

File: site/src/pages/release-notes/1.20.2.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  when an <type://HttpResponse> is correctly sent. #4494
+- <type://DocService> is correctly generated for a <type://GrpcService> when
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.20.2.mdx:11]:
```suggestion
- <type://DocService> is correctly generated for a decorated <type://GrpcService> when
```

**Review by jrhee17 — APPROVED** [review]
Thanks @minwoox ! 🙇 👍 🚀

**Comment by minwoox** [comment]
screencapture-localhost-8000-release-notes-1-20-2-2022-11-07-15_44_33

**Comment by minwoox** [comment]
screencapture-localhost-8000-release-notes-1-20-2-2022-11-07-15_46_56

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox!

**Comment by minwoox** [comment]
Thanks for reviewing. 😉

**Pull request merged**